src/test/ui/modules_and_files_visibility/mod_file_disambig.stderr

```diff
@@ -1,4 +1,4 @@
-error[E0761]: file for module `mod_file_disambig_aux` found at both mod_file_disambig_aux.rs and mod_file_disambig_aux/mod.rs
+error[E0761]: file for module `mod_file_disambig_aux` found at both "mod_file_disambig_aux.rs" and "mod_file_disambig_aux/mod.rs"
```

Comment:
@Zerotask stated that it should be:

>      error[E0761]: file for module `user` found at both "src\user.rs" and "src\user\mod.rs"

but this doesn't include `src`.
What should it be?

Reply:
Oops! I missed it. Let me take a look.

Reply:
Added.